MapboxCoreNavigation/CLHeading.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import CoreLocation
+
+extension CLHeading {
+    var preferredHeading: CLLocationDirection {
+        guard trueHeading <= 0 || headingAccuracy > 45 else {
+            return magneticHeading
+        }
+        return trueHeading
```

Comment:
I’m a bit confused by the intent of this computed variable: we return `trueHeading` if it’s negative or `headingAccuracy` is really high (really inaccurate)?